net/netavark/Makefile

```
@@ -34,18 +33,24 @@ define Package/netavark/description
 endef
 
 define Package/netavark/conffiles
-/etc/config/netavark
+  /etc/config/netavark
```

```suggestion
/etc/config/netavark
```

indentation is not used for conffiles

---

I'll fix it..